File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/test/ConfirmDialogRenderer.java
package com.github.kotvertolet.youtubejextractor.models.test;

import java.util.List;
import com.google.gson.annotations.SerializedName;

public class ConfirmDialogRenderer{

	@SerializedName("trackingParams")
	private String trackingParams;

	@SerializedName("cancelButton")
	private CancelButton cancelButton;

	@SerializedName("confirmButton")
	private ConfirmButton confirmButton;

	@SerializedName("dialogMessages")
	private List<DialogMessagesItem> dialogMessages;

	@SerializedName("primaryIsCancel")
	private boolean primaryIsCancel;

	public void setTrackingParams(String trackingParams){
		this.trackingParams = trackingParams;
	}

	public String getTrackingParams(){
		return trackingParams;
	}

	public void setCancelButton(CancelButton cancelButton){
		this.cancelButton = cancelButton;
	}

	public CancelButton getCancelButton(){
		return cancelButton;
	}

	public void setConfirmButton(ConfirmButton confirmButton){
		this.confirmButton = confirmButton;
	}

	public ConfirmButton getConfirmButton(){
		return confirmButton;
	}

	public void setDialogMessages(List<DialogMessagesItem> dialogMessages){
		this.dialogMessages = dialogMessages;
	}

	public List<DialogMessagesItem> getDialogMessages(){
		return dialogMessages;
	}

	public void setPrimaryIsCancel(boolean primaryIsCancel){
		this.primaryIsCancel = primaryIsCancel;
	}

	public boolean isPrimaryIsCancel(){
		return primaryIsCancel;
	}
}

File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/newModels/ConfirmDialogRenderer.kt
package com.github.kotvertolet.youtubejextractor.models.newModels

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize
import java.io.Serializable

@Parcelize
class ConfirmDialogRenderer : Parcelable, Serializable {
    @SerializedName("trackingParams")
    var trackingParams: String? = null

    @SerializedName("cancelButton")
    var cancelButton: CancelButton? = null

    @SerializedName("confirmButton")
    var confirmButton: ConfirmButton? = null

    @SerializedName("dialogMessages")
    var dialogMessages: List<DialogMessagesItem>? = null

    @SerializedName("primaryIsCancel")
    var isPrimaryIsCancel = false
    override fun toString(): String {
        return "ConfirmDialogRenderer{" +
                "trackingParams = '" + trackingParams + '\'' +
                ",cancelButton = '" + cancelButton + '\'' +
                ",confirmButton = '" + confirmButton + '\'' +
                ",dialogMessages = '" + dialogMessages + '\'' +
                ",primaryIsCancel = '" + isPrimaryIsCancel + '\'' +
                "}"
    }
}